RecoTracker/TkDetLayers/src/Phase2EndcapRingBuilder.cc

```diff
@@ -1,5 +1,6 @@
 #include "Phase2EndcapRingBuilder.h"
 #include "TrackingTools/DetLayers/interface/DetLayerException.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
```

Comment:
Does this file even belong in this PR? It doesn't seem related, and it brings in the requirement for a Reco signature.

Reply:
@cvuosalo unfortunately yes, without it the code does not compile, you may easily try yourself.